flask_of_oil/oauth_filter.py

```diff
@@ -61,8 +61,12 @@ def configure_with_multiple_jwt_issuers(self, issuers, audience, scopes=None):
             self.scopes = scopes
 
         for issuer in issuers:
-            jwks_url = issuer + "/jwks"
-            self.validators[issuer] = JwtValidator(jwks_url, issuer, audience, self.verify_ssl)
+            if isinstance(issuer, str):
+                jwks_url = issuer + "/jwks"
+                self.validators[issuer] = JwtValidator(jwks_url, issuer, audience, self.verify_ssl)
+            elif isinstance(issuer, dict):
+                self.validators[issuer['name']] = JwtValidator(issuer['url'], issuer['name'], issuer['audience'],
```

Comment:
what says the issuer is in the `name` entry of the dictionary? Similar question for `url` and `audience`.